src/test/java/jalau/at18/katas/phonenumber/mauricio/MainTest.java

```diff
@@ -0,0 +1,29 @@
+package jalau.at18.katas.phonenumber.mauricio;
+
+import static org.junit.Assert.*;
+import org.junit.Test;
+
+public class MainTest {
+    private static final int ZERO = 0;
+    private static final int ONE = 1;
+    private static final int TWO = 2;
+    private static final int THREE = 3;
+    private static final int FOUR = 4;
+    private static final int FIVE = 5;
+    private static final int SIX = 6;
+    private static final int SEVEN = 7;
+    private static final int EIGHT = 8;
+    private static final int NINE = 9;
+    private static final int FOURTEEN = 14;
+
+    @Test
+    public void mainShouldSayHello() {
+        int[] numberList = {ZERO, ONE, TWO, THREE, FOUR, FIVE, SIX, SEVEN, EIGHT, NINE};
+        int phoneNumberLengthExpected = FOURTEEN;
+
+        Main main = new Main();
+
+        int phoneNumberLengthObtained = main.phoneNumber(numberList).length();
```

Comment:
Verifying result length is ok, but in this case, the structure of the result string is also important for kata requirement, in this case, your test is vulnerable to do not catch issues if the code is broken, for example, if code returns 
`(123X-456-7890`
test will still pass, but that is wrong because value above is incorrect for kata requirement, so test should fail. In order to fix that, please update the test so it validates not only length, but actual result expected based on kata requirement